pkg/cmd/admin/policy/reconcile_clusterroles.go

```diff
@@ -97,6 +99,19 @@ func (o *ReconcileClusterRolesOptions) Complete(cmd *cobra.Command, f *clientcmd
 
 	o.Output = kcmdutil.GetFlagString(cmd, "output")
 
+	mapper, _ := f.Object()
+	for _, resourceString := range args {
+		resource, name, err := osutil.ResolveResource("clusterroles", resourceString, mapper)
+		if err != nil {
+			return err
+		}
+		if resource != "clusterroles" {
+			return fmt.Errorf("%s is not a valid resource type for this command", resource)
+		}
+
+		o.RolesToReconcile = append(o.RolesToReconcile, name)
+	}
+
```

Comment:
should we validate any explicitly requested roles actually exist in `bootstrappolicy.GetBootstrapClusterRoles()`?